include/internal/SampleBufferData.h

```diff
@@ -0,0 +1,78 @@
+#ifndef LMMS_SAMPLEBUFFERDATA_H
+#define LMMS_SAMPLEBUFFERDATA_H
+
+#include <vector>
+#include <QtCore/QString>
+#include "Mixer.h"
+#include "Engine.h"
+#include "MemoryManager.h"
+#include "lmms_basics.h"
+
+namespace internal {
+	sample_rate_t GetMixerSampleRate();
```

Comment:
Function names should be `camelCase` starting with the lower case. Also, please consider removing the `get` prefix here if it's not necessary.

Reply:
Thanks! I got confused with Google's code style.